tests/ui/modules/module_suggestion_when_module_not_found/success.rs

```diff
@@ -0,0 +1,4 @@
+//@ ignore-auxiliary
+
+use submodule3::ferris; //~ ERROR
```

Comment:
```suggestion
use submodule3::ferris;
```
These annotations shouldn't be used if there's no error (you can write a regular comment saying that this module does not actually exist).